installation/README.md

```diff
@@ -1,9 +1,102 @@
 # ROS2 Installation
 
-This scripts allows you to easily install ROS2 on Debian Buster. Note that this script is in very early
-stage and it is not tested yet.
+Currently, we provide two ways to install ROS2 and `epuck_ros2` e-puck:
+- [Easy and recommended](#using-pi-puck-image): Using the given SD card image with ROS2 and `epuck_ros2` preinstalled.
+- [Hard](#fresh-ros2-installation): Fresh ROS2 installation on top of [Raspberry Pi OS (based on Debian Buster)](https://www.raspberrypi.org/downloads/raspberry-pi-os/).
 
-## Configuration
+---
+
+## Using Pi-puck Image
+This is the recommended way of installing ROS2 on Pi-puck as all ROS2 compilation issues are resolved by us.
+Moreover, this is very fast as you don't have to wait for the compilation process to finish.
+It is enough to download our image and burn it to SD card that has to be at least 8GB.
+To burn the image on the SD card you can use the following command on Linux:
+```bash
+dd bs=4M if=path_to_pi_puck_image.img of=/dev/sdX conv=fsync
+```
+It is very important to correct the output device, it is usually named as `/dev/sdb` (not `/dev/sdb1`).
+To check whether the block device is the one associated with the SD card you can run before inserting the SD card `ls /dev/sd*`, then insert the SD card and run the same command again and compare the output.
+For, more detailed instructions on how to burn an image to the SD card for different OS' you can check the following links:
+
+> Detailed instructions about burning SD card for 
+[Windows](https://www.raspberrypi.org/documentation/installation/installing-images/windows.md),
+[Mac](https://www.raspberrypi.org/documentation/installation/installing-images/mac.md) and 
+[Linux](https://www.raspberrypi.org/documentation/installation/installing-images/linux.md)
+
+### Wifi and SSH
+Once the image is burned on SD card you may want to configure wifi connection.
+By default, Pi-puck with the provided image will try to connect to a wifi network with SSID `E-PUCK`.
+If you want to change the default behaviour or to add more networks you can add the networks to `/etc/wpa_supplicant/wpa_supplicant.conf` (file located on the SD card).
+For example:
+```conf
+network={
+    ssid="my_network_ssid"
+    psk="my_network_password"
+    key_mgmt=WPA-PSK
+}
+```
+> Detailed instructions on wifi configuration can be found [here](https://www.raspberrypi.org/documentation/configuration/wireless/wireless-cli.md).
+
+At this point, you probably want to start the nodes from `epuck_ros2`.
+First, make sure that your pi-puck and computer are connected to the same local network.
```

Comment:
The spelling of pi-puck is not consistent across this documentation, it is sometimes Pi-puck, sometimes pi-puck.